src/partitionAccessory.ts

```diff
@@ -129,6 +129,7 @@ export class EnvisalinkPartitionAccessory {
             case 'disarmed':
             case 'notready':
             case 'failedarm':
```

Comment:
Let's remove `failedarm`. That status is not present in [NodeAlarmProxy](https://github.com/dustindclark/NodeAlarmProxy/blob/49d817d3503740911eb54ac8ccade817a56554b8/envisalink.js#L474).

Reply:
Done.